app/views/Tagging/SimplifiedTextView/index.tsx

```diff
@@ -0,0 +1,193 @@
+import React from 'react';
+import { IoAddCircle } from 'react-icons/io5';
+import { useTextSelection } from 'use-text-selection';
+import { _cs } from '@togglecorp/fujs';
+import { Button } from '@the-deep/deep-ui';
+
+import TaggedExcerpt from './TaggedExcerpt';
+import styles from './styles.css';
+
+export interface Entry {
+    clientId: string;
+    droppedExcerpt: string;
+    excerpt: string;
+}
+
+interface Split {
+    startIndex: number;
+    endIndex: number;
+    excerpt: string;
+    droppedExcerpt: string;
+}
+
+function filterOutNull<D, FD extends Exclude<D, null>>(list: D[] | undefined): FD[] {
+    if (!list) {
+        return [];
+    }
+
+    return list.filter((d) => !!d) as FD[];
+}
```

Comment:
This does not just filter out null.
This will filter out `false`, `null`, `undefined`, `0`, `''`.